lib/Common/Memory/Recycler.inl

```diff
@@ -36,18 +36,22 @@ namespace Memory
 class DummyVTableObject : public FinalizableObject
 {
 public:
-    virtual void Finalize(bool isShutdown) {}
-    virtual void Dispose(bool isShutdown) {}
-    virtual void Mark(Recycler * recycler) {}
+    virtual void __stdcall Finalize(bool isShutdown) {}
+    virtual void __stdcall Dispose(bool isShutdown) {}
+    virtual void __stdcall Mark(Recycler * recycler) {}
 };
 }
 
 template <ObjectInfoBits attributes, bool nothrow>
 inline char *
 Recycler::AllocWithAttributesInlined(DECLSPEC_GUARD_OVERFLOW size_t size)
 {
-    // All tracked objects are client tracked objects
+    // All tracked objects are client tracked or recycler host visited objects
```

Comment:
Confused by this comment- why are host visited objects allocated with the track bit?

Reply:
We are currently using the TrackBit to indicate an object that needs tracing.  All our RecyclerVisitedHostTraced* objects currently have the TrackBit set.  Seems like we could have chosen to use "not Leaf" for the same purpose.  I'll explore that option.